migrations/versions/cdbd609e2bd5_.py

```diff
@@ -0,0 +1,37 @@
+"""
+1.3.1 - Rename biospecimen table column volume_ml to volume_ul
+
+Revision ID: cdbd609e2bd5
+Revises: ce01ada6ac80
+Create Date: 2018-08-27 10:28:59.687358
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+import dataservice
+
+# revision identifiers, used by Alembic.
+revision = 'cdbd609e2bd5'
+down_revision = 'ce01ada6ac80'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('biospecimen', sa.Column(
+        'volume_ul', sa.Float(), nullable=True))
+    op.drop_column('biospecimen', 'volume_ml')
+    op.add_column('biospecimen_diagnosis', sa.Column(
```

Comment:
This looks like it maybe shouldn't be here? I thought you made that change to the model in another PR we already merged